adlik_serving/runtime/openvino/model/ir_model.cc

```diff
@@ -262,8 +262,12 @@ tensorflow::Status IRModel::init() {
     network.setBatchSize(config.max_batch_size());
   }
   // 4. set instance
-  ExecutableNetwork executableNetwork = core.LoadNetwork(network, "CPU");
   for (const auto& group : config.instance_group()) {
+    ExecutableNetwork executableNetwork;
+    if (group.kind() != adlik::serving::ModelInstanceGroup::KIND_GPU)
```

Comment:
alwyas need `{}` in `if-else` statement to make code clear to read and avoid some hidden bugs.